eq-author/src/components/CommentsPanel/index.test.js

```diff
@@ -194,4 +286,105 @@ describe("Comments Pane", () => {
     expect(queryWasCalled).toBeTruthy();
     expect(getByText("Oops! Something went wrong")).toBeTruthy();
   });
+
+  it("should render disabled delete button if comments exist && comment user !== me", async () => {
+    const { getByTestId } = renderWithContext(<CommentsPanel />, { ...props });
+
+    await flushPromises();
+    const deleteCommentButton = getByTestId("btn-delete-comment-0");
+
+    expect(queryWasCalled).toBeTruthy();
+    expect(deleteCommentButton).toHaveAttribute("disabled");
+  });
+
+  it("should be able to delete an existing comment", async () => {
+    const { getByText, getByTestId } = renderWithContext(<CommentsPanel />, {
+      ...props,
+    });
+
+    await flushPromises();
+
+    const comment2 = getByText("Query comment2 body");
+    expect(comment2).toBeTruthy();
+
+    fireEvent.click(getByTestId("btn-delete-comment-1"));
+    await flushPromises();
+
+    expect(queryWasCalled).toBeTruthy();
+    expect(comment2).toBeTruthy();
+  });
+
+  it("should render disabled Edit button if comments exist && comment user !== me", async () => {
+    const { getByTestId } = renderWithContext(<CommentsPanel />, { ...props });
+
+    await flushPromises();
+    const editCommentButton = getByTestId("btn-edit-comment-0");
+
+    expect(queryWasCalled).toBeTruthy();
+    expect(editCommentButton).toHaveAttribute("disabled");
+  });
+
+  it("should render enabled Edit button if comments exist && comment user === me", async () => {
+    const { getByTestId } = renderWithContext(<CommentsPanel />, { ...props });
+
+    await flushPromises();
+    const editCommentButton = getByTestId("btn-edit-comment-1");
+
+    expect(queryWasCalled).toBeTruthy();
+    expect(editCommentButton).not.toHaveAttribute("disabled");
+  });
+
+  // it("should focus on relevant textarea and enable editing when edit button is clicked", async () => {
+  //   const { getByTestId } = renderWithContext(<CommentsPanel />, { ...props });
+
+  //   await flushPromises();
+  //   fireEvent.click(getByTestId("btn-edit-comment-1"));
+
+  //   const editCommentTxtArea = getByTestId("edit-comment-txtArea-1");
+
+  //   expect(queryWasCalled).toBeTruthy();
+  //   expect(editCommentTxtArea.disabled).toBeFalsy();
+  //   expect(editCommentTxtArea).toHaveFocus();
+  // });
```

Comment:
I think it's best to just drop this till we figure it out.

Reply:
removed